tests/sockets/sockets.c

```diff
@@ -260,13 +281,16 @@ void test_sockets(args_t* args)
     pthread_t srv_thread;
     pthread_t cli_thread;
 
+    pthread_barrier_init(&bar, NULL, 2);
+
     assert(pthread_create(&srv_thread, NULL, _srv_thread_func, args) == 0);
-    _sleep_msec(100);
```

Comment:
Why the sleep is no longer needed?

Reply:
I changed the test to use a `pthread_barrier_t`.